proto/data.go

```diff
@@ -636,6 +636,8 @@ func (ms *MVCCStats) Add(oms *MVCCStats) {
 	ms.IntentCount += oms.IntentCount
 	ms.IntentAge += oms.IntentAge
 	ms.GCBytesAge += oms.GCBytesAge
+	ms.SysBytes += oms.SysBytes
+	ms.SysCount += oms.SysCount
 	ms.LastUpdateNanos += oms.LastUpdateNanos
```

Comment:
I noticed this earlier; we're not treating LastUpdateNanos correctly, I don't think it's simply a matter of addition.